src/derive/derive_K.H

```diff
@@ -10,53 +10,81 @@ namespace amr_wind {
 struct StencilInterior
 {
     // First derivatives
-    static constexpr amrex::Real c00 =  0.5;
-    static constexpr amrex::Real c01 =  0.0;
-    static constexpr amrex::Real c02 = -0.5;
-    static constexpr amrex::Real c10 =  c00;
-    static constexpr amrex::Real c11 =  c01;
-    static constexpr amrex::Real c12 =  c02;
-    static constexpr amrex::Real c20 =  c00;
-    static constexpr amrex::Real c21 =  c01;
-    static constexpr amrex::Real c22 =  c02;
+    // fx= (ax*f_{i+1,j,k}+bx*f_{i,j,k}+cx*f_{i-1,j,k})/dx
+    // fy= (ay*f_{i,j+1,k}+by*f_{i,j,k}+cy*f_{i,j-1,k})/dy
+    // fz= (az*f_{i,j,k+1}+bz*f_{i,j,k}+cz*f_{i,j,k-1})/dz
+    static constexpr amrex::Real c00 =  0.5; // ax 
+    static constexpr amrex::Real c01 =  0.0; // bx
+    static constexpr amrex::Real c02 = -0.5; // cx
+    static constexpr amrex::Real c10 =  c00; // ay
+    static constexpr amrex::Real c11 =  c01; // by
+    static constexpr amrex::Real c12 =  c02; // cy
+    static constexpr amrex::Real c20 =  c00; // az
+    static constexpr amrex::Real c21 =  c01; // bz
+    static constexpr amrex::Real c22 =  c02; // cz
 
     // Second derivatives
-    static constexpr amrex::Real s00 =  1.0;
-    static constexpr amrex::Real s01 = -2.0;
-    static constexpr amrex::Real s02 =  1.0;
-    static constexpr amrex::Real s10 =  s00;
-    static constexpr amrex::Real s11 =  s01;
-    static constexpr amrex::Real s12 =  s02;
-    static constexpr amrex::Real s20 =  s00;
-    static constexpr amrex::Real s21 =  s01;
-    static constexpr amrex::Real s22 =  s02;
+    // fxx= (axx*f_{i+1,j,k}+bxx*f_{i,j,k}+cxx*f_{i-1,j,k})/dx^2
+    // fyy= (ayy*f_{i,j+1,k}+byy*f_{i,j,k}+cyy*f_{i,j-1,k})/dy^2
+    // fzz= (azz*f_{i,j,k+1}+bzz*f_{i,j,k}+czz*f_{i,j,k-1})/dz^2
+    static constexpr amrex::Real s00 =  1.0; // axx 
+    static constexpr amrex::Real s01 = -2.0; // bxx
+    static constexpr amrex::Real s02 =  1.0; // cxx
+    static constexpr amrex::Real s10 =  s00; // ayy
+    static constexpr amrex::Real s11 =  s01; // byy
+    static constexpr amrex::Real s12 =  s02; // cyy
+    static constexpr amrex::Real s20 =  s00; // azz
+    static constexpr amrex::Real s21 =  s01; // bzz
+    static constexpr amrex::Real s22 =  s02; // czz
 
 };
 
 struct StencilILO
 {
-    static constexpr amrex::Real c00 =  0.5;
-    static constexpr amrex::Real c01 =  0.5;
-    static constexpr amrex::Real c02 = -1.0;
+    static constexpr amrex::Real c00 =  1./3.;
+    static constexpr amrex::Real c01 =  1.;
+    static constexpr amrex::Real c02 = -4./3.;
```

Comment:
So looks like this is going to cause diffs in ABL regression tests. Have you looked at how much of a difference this causes?

Reply:
```test 7
    Start  7: abl_godunov

7: Test command: /bin/sh "-c" "/usr/local/bin/mpiexec -n 4  /Users/mbrazell/Codes/amr-wind/build_mac/amr_wind  /Users/mbrazell/Codes/amr-wind/build_mac/test/test_files/abl_godunov/abl_godunov.i time.max_step=10 amr.plot_file=plt amr.checkpoint_files_output=0 amr.plot_files_output=1 amrex.throw_exception=1 amrex.signal_handling=0 > abl_godunov.log && /Users/mbrazell/Codes/amr-wind/build_mac/submods/amrex/Tools/Plotfile/fcompare /Users/mbrazell/Codes/amr-wind/test/AMR-WindGoldFiles/Darwin/AppleClang/11.0.0.11000033/abl_godunov/plt00010 /Users/mbrazell/Codes/amr-wind/build_mac/test/test_files/abl_godunov/plt00010"
7: Test timeout computed to be: 5400
7: 
7:             variable name            absolute error            relative error
7:                                         (||A - B||)         (||A - B||/||A||)
7:  ----------------------------------------------------------------------------
7:  level = 0
7:  density                                          0                         0
7:  gpx                                 1.92758915e-06           0.0001293333767
7:  gpy                                 2.02401295e-06           0.0001291563875
7:  gpz                                5.298073561e-06             1.8539462e-05
7:  mu_turb                              0.02877076275             0.07270538018
7:  p                                  0.0001082316827           1.770245591e-06
7:  temperature                        5.684341886e-13           1.841268755e-15
7:  velocityx                          7.598550437e-05           1.074478421e-05
7:  velocityy                          7.426268384e-05           1.220242844e-05
7:  velocityz                          7.033488545e-06           0.0001706259052
7:  velocity_mueff                       0.02877076275             0.07270354292```